### PR TITLE
arb-ts: only use networks config values both both networks are configged 

### PR DESCRIPTION
**Pull request by DZGoldman**

### CODE REVIEW

**Comment by DZGoldman** [comment]
(one more thing to push)

File: packages/arb-bridge-peripherals/scripts/deploy_token_bridge_l1.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ const main = async () => {
   const l2SignerAddress = await l2Signer.getAddress()
 
   // parse needed vars
-  const inboxAddress = l1Network.tokenBridge.inbox
+  const inboxAddress = await bridge.l1Bridge.getInbox()
   if (!inboxAddress) throw new Error('Please set inbox address! INBOX_ADDRESS')
```
Comment (hkalodner) [inline packages/arb-bridge-peripherals/scripts/deploy_token_bridge_l1.ts:36]: I think this check no longer makes sense? Idk if we want to replace with a check for `bridge.l1Bridge` above the `getInbox` call

**Review by hkalodner — CHANGES_REQUESTED** [review]

File: packages/arb-bridge-peripherals/scripts/deploy_token_bridge_l1.ts (hunk 1 above)
Comment (DZGoldman) [reply packages/arb-bridge-peripherals/scripts/deploy_token_bridge_l1.ts:36]: ye we can just remove it entirely; `.l1Bridge` will always be present, and  `getInbox()` will itself throw when appropriate

**Review by hkalodner — APPROVED** [review]
LGTM

**Pull request merged**